README.md

```diff
@@ -24,6 +24,26 @@ does.  Chrome on Linux will support it with some fontconfig tweaking, see
 [issue #36](https://github.com/googlei18n/noto-emoji/issues/36). Currently we do
 not build other color font formats.
 
+## Color emoji assets
+
+The assets provided in the repo are all those used to build the NotoColorEmoji
+font.  Note however that NotoColorEmoji often uses the same assets to represent
+different character sequences-- notably, most gender-neutral characters or
+sequences are represented using assets named after one of the gendered
+sequences.  This means that some sequences appear to be missing.  Definitions of
+the aliasing used appear in the aliases.txt file.
```

Comment:
Shouldn't that be `emoji_aliases.txt`?

Reply:
Yep, rushed it. Thanks for pointing that out.